kong/cmd/reload.lua

```diff
@@ -65,6 +66,7 @@ Options:
  -c,--conf        (optional string) configuration file
  -p,--prefix      (optional string) prefix Kong is running at
  --nginx-conf     (optional string) custom Nginx configuration template
+ --test-flags     (optional string)
```

Comment:
If this shows in CLI output, should it at least have some description? Is it also meant for endusers?

Reply:
agree on adding a short description. I would also like to rename this to something more appropriate like `--conf-template-vars`